clinicadl/clinicadl/tools/data/generate_data.py

```diff
@@ -161,6 +161,10 @@ def generate_trivial_dataset(caps_dir, output_dir, n_subjects, tsv_path=None, pr
         else:
             mask_path = join(cache_clinicadl, 'AAL2')
 
+    # Create subjects dir
+    if not exists(join(output_dir, 'subjects')):
+        makedirs(join(output_dir, 'subjects'))
```

Comment:
To avoid race condition when creating, it would be better to use `try..catch`or, for this function, the [`exist_ok` option](https://docs.python.org/3/library/os.html#os.makedirs):
```suggestion
    os.makedirs(join(output_dir, 'subjects'), exist_ok=True)
```